src/_P092_DLbus.ino

```diff
@@ -934,8 +934,8 @@ boolean P092_fetch_heatmeter(int number, sP092_ReadData* ReadData) {
   if (HMindex.IndexIsValid == 0)
     return false;
   heat_meter = (DLbus_Data->ByteStream[HMindex.kwh_index + 1] << 8) | DLbus_Data->ByteStream[HMindex.kwh_index];
-  heat_meter_mwh = (heat_meter * 0.1) / 1000; // in MWh
-  if (heat_meter_mwh > 1.0) {
+  heat_meter_mwh = (heat_meter * 0.1f) / 1000f; // in MWh
```

Comment:
The compiler doesn't appreciate that, expects 1000.0f or just 1000

Reply:
Ah, had already fixed a number of those incorrect replaces I made.
Apparently I did not compile with this plugin.
Thanks for noticing.